code/modules/mob/living/basic/heretic/maid_in_the_mirror.dm

```diff
@@ -16,36 +16,32 @@
 	melee_damage_upper = 16
 	sight = SEE_MOBS | SEE_OBJS | SEE_TURFS
 	death_message = "shatters and vanishes, releasing a gust of cold air."
-	loot = list(
-		/obj/item/shard,
+	/// How often being examined by a specific mob can hurt us
+	var/recent_examine_damage_cooldown = 10 SECONDS
+	/// A list of REFs to people who recently examined us
+	var/list/recent_examiner_refs = list()
+
+/mob/living/basic/heretic_summon/Initialize(mapload)
+	. = ..()
+	var/static/list/loot = list(
 		/obj/effect/decal/cleanable/ash,
 		/obj/item/clothing/suit/armor/vest,
 		/obj/item/organ/internal/lungs,
+		/obj/item/shard,
 	)
-	actions_to_add = list(/datum/action/cooldown/spell/jaunt/mirror_walk)
+	AddElement(/datum/element/death_drops, loot)
+	var/datum/action/cooldown/spell/jaunt/mirror_walk/jaunt = new (src)
+	jaunt.Grant(src)
 
-	/// Whether we take damage when we're examined
-	var/weak_on_examine = TRUE
```

Comment:
I added this var so admins could disable it with VV for event stuff, could you keep it?